content/topics/ENA-submission-tutorial.md

```diff
@@ -0,0 +1,473 @@
+        <br><br>
+        In the sample sheet, the first row contains the following:
+        <ol>
+          <li>The name of the checklist</li>
+          <li>Field names</li>
+          <li>Description of the field</li>
+          <li>If the field is mandatory, recommended or optional</li>
+        </ol>
+        Note that some fields have controlled vocabulary, which are available in the template as drop-down lists (the lists become visible when you click on a cell).
+        <br><br>
+        <b>Note</b>: It is <i>strongly</i> recommended that you provide as much information as possible in the metadata sheets as it will increase the <a href="https://www.go-fair.org/fair-principles/" target="_blank">FAIRness</a> of your data, and thus the probability of it being useful in future research efforts. 
+        <h3>An example</h3>
+        In this scenario we have used the Tree of life checklist. A completed template is found in <a href="/static/files/ena_tutorial/Alectoris-graeca-metadata.xlsx">Alectoris-graeca-metadata.xlsx</a>, adapted from a real submission made under the <a href="https://biodiversitygenomics.eu/" target="_blank">Biodiversity Genomics Europe</a> (BGE) project organised by the <a href="https://www.erga-biodiversity.eu/" target="_blank">European Reference Genome Atlas</a> (ERGA) initiative. The task is to submit four datasets (one PacBio HiFi WGS, two Illumina HiC, and one Illumina RNAseq), including four samples, all in one study, plus a chromosome level assembly in another study, and then group these them studies together under an umbrella study. Please note that it is not necessary to submit the assembly in a study separate from the raw reads. In our example it is done to demonstrate how umbrella studies can be used.
+    </span>
+  </div>  
+  <br>
+</div>
+
+<p>
+  <button class="btn btn-expandable" data-bs-toggle="collapse" href="#collapseFind3" role="button" aria-expanded="false" aria-controls="collapseFind3">
+    Preparing the data files
+    <i class="bi bi-chevron-double-down p-2"></i>
+  </button>
+</p>
+<div class="collapse" id="collapseFind3">
+  <div class="card card-body">
+    <span>
+    In the run sheet of a metadata template, the file names of the raw reads and their md5 checksums are collected. For the three types of submissions of raw reads described in this tutorial, the data files need to be uploaded to the dropbox on ENA's servers. The exact name of the files as they reside at ENA, including any subfolders, should be filled in the run sheet. The checksums are used to validate that the data transfers have been completed without loss.
+    <h3>Checksums</h3>
+    Typically you will receive the checksums from the data producer in the delivery, but in case you need to calculate them yourself:
```

Comment:
Some researchers may not be familiar with checksums. Would it be use ful to add a link to for example the wikipeda article about checksum (or some other brief introduction to the concept)?